.github/workflows/markdownlint.yml

```diff
@@ -28,4 +26,3 @@ jobs:
       with:
         use-quiet-mode: 'yes'
         use-verbose-mode: 'no'
```

Comment:
you need to keep the config-file: this contains the exceptions like ./link-to-the-work-item - this is why you are getting the link error in your PR.

Reply:
Yepp, reverted back